docs/docs/1.1.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+
+Cross-zone communication between services rqeuires mTLS because Ingress routes connections with SNI.
+
+### Apply TrafficPermission policy
+
+You must also apply a [TrafficPermission policy](../policies/traffic-permissions.md). Traffic permissions also require enabling mTLS. Kuma creates a default `TrafficPermission` policy that allows all the communication between all the services when a new `Mesh` is created. You should take care to create appropriate policies to restrict traffic between services. 
+
+### Ingress requirements
+
+Cross-zone communication between services is available only if Ingress has a public address and public port.
+
+On Kubernetes, Kuma automatically tries to pick up the public address and port. Depending your load balancing implementation, you might need to wait a few minutes for Kuma to get the address. 
+
+### Cross-communication details
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+To view the list of service names available for cross-zone communication, run:
+
+```bash
+$ kubectl get dataplanes -n echo-example -o yaml | grep service
+           service: echo-server_echo-example_svc_1010
+```
+
+To consume the example service only within the same Kuma zone, you can run:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server:1010
+```
+
+To consume the example service across all zones in your Kuma deployment (that is, from endpoints ultimately connecting to the same global control plane), you can run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh:80
+```
+
+And if your HTTP clients take the standard default port 80, you can the port value and run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh
+```
+
+Because Kuma on Kubernetes relies on transparent proxy, `kuma-dp` listens on port 80 for all virtual IPs that are assigned to services in the `.mesh` DNS zone. The DNS names are rendered RFC compatible by replacing underscores with dots.
+
+:::
+::: tab "Universal"
+
+With a hybrid deployment, running in both Kubernetes and Universal mode, the service tag should be the same in both environments (e.g `echo-server_echo-example_svc_1010`):
+
+```yaml
+type: Dataplane
+mesh: default
+name: backend-02 
+networking:
+  address: 127.0.0.1
+  inbound:
+  - port: 2010
+    servicePort: 1010
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+With a Universal control plane, the service tag has no such limitation.
+
+REVIEWER: DOES THIS ^ APPLY TO A HYBRID ENVIRONMENT WHERE THE CONTROL PLANE RUNS IN UNIVERSAL MODE? DOES THIS REFER TO THE GLOBAL CONTROL PLANE? CAN WE CLARIFY THE DISTINCTION BETWEEN THESE TWO CASES (THAT IS, THE HYBRID AND THIS SECOND CASE WHICH MIGHT OR MIGHT NOT BE RELEVANT TO THE FIRST ONE)?
```

Comment:
If there is a chance that the service is deployed on K8s that limitation will always apply. So let's try to enumerate here, which use-cases will have the limitation:
* all zones are K8s 
* hybrid mode, at least one zone is K8s. The service runs on that K8s zone(s)

Use-cases where the limitation does not apply:
* all zones are Universal
* hybrid mode, where the service runs ONLY in a universal zone (or many universal zones)